feat: add send from to the confirm send screen

reallybeard opened the pull request:
## Description

- Reuse our new `AccountDropdown` component to display what account you are sending from

## Notice

<!-- Before submitting a pull request, please make sure you have answered the following: -->

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

n/a

## Risk

this should pose no risk

## Testing

Confirm that the account that is displayed is accurate

### Engineering

Implemented this the same way we do for the details view, so should act in the same way.

### Operations

On the confirm screen of send, verify that the account shown is the correct account.

## Screenshots (if applicable)
![Screen Shot 2022-09-13 at 5 27 13 PM](https://user-images.githubusercontent.com/89934888/190021246-901f4096-a70a-4f30-95fa-0eeb25311a83.png)

gomesalexandre commented on src/components/Modals/Send/views/Confirm.tsx:
```diff
@@ -56,6 +65,9 @@ export const Confirm = () => {
   const borderColor = useColorModeValue('gray.100', 'gray.750')
 
+  // We don't want this firing -- but need it for typing
+  const handleAccountChange = () => {}
```
Not for this PR, but leaving this here for reference: we should make a stronger discriminated union in `<AccountDropdown />` so you don't need to pass this down when `disabled` and a `defaultAccountId` is passed down, which means `<AccountDropdown />` is effectively controlled at this point thus we don't need to handle the account change

gomesalexandre reviewed (changes requested):
Tested locally, functionally LGTM for UTXOs and EVMs

Spotted two issues here:

1. Alignment looks a bit off because of `<AccountDropdown />` padding, do we want to remove padding?
2. This isn't working for Cosmos SDK and the same logic will need to be added in `src/plugins/cosmos/components/modals/Send/views/Confirm.tsx`

reallybeard commented:
> Tested locally, functionally LGTM for UTXOs and EVMs
> 
> Spotted two issues here:
> 
> 1. Alignment looks a bit off because of `<AccountDropdown />` padding, do we want to remove padding?
> 2. This isn't working for Cosmos SDK and the same logic will need to be added in `src/plugins/cosmos/components/modals/Send/views/Confirm.tsx`

Fixed alignment, and added to cosmos.

![Screen Shot 2022-09-14 at 9 50 08 AM](https://user-images.githubusercontent.com/89934888/190188278-01d954a7-6979-4235-8c20-de9c73c793d2.png)

gomesalexandre reviewed (approved):
Tested again, the `Send From` row is properly matching the selected account for EVM (tested with KK), UTXOs, and Cosmos accounts.

The selected account is now using ghost variant with 0 padding, which makes it properly aligned with the other rows

<img width="513" alt="image" src="https://user-images.githubusercontent.com/17035424/190191736-6cfc97e1-0fc4-4b92-8a3e-53b52a77b93c.png">

🍬
